taskcluster/docker/node/build.py

```diff
@@ -127,7 +144,7 @@ def mkdir(path):
 def get_hash(path, hash_alg="sha256"):
     h = hashlib.new(hash_alg)
     with open(path, "rb") as fh:
-        for chunk in iter(functools.partial(fh.read, 4096), b''):
+        for chunk in iter(functools.partial(fh.read, 4096), b""):
```

Comment:
The changes below this line are just formatting changes caused by running `black` on this file.